init-ci.ps1

```diff
@@ -80,6 +81,24 @@ if ([string]::IsNullOrEmpty($DemoTeamRegistry)) {
     throw
   }
 }
+
+if ([string]::IsNullOrEmpty($SitecoreRegistry)) {
+  # if it wasn't passed as a parameter, let's try to find it in environment
+  $SitecoreRegistry = $env:INTERNAL_SITECORE_DOCKER_REGISTRY
+  if ($null -eq $SitecoreRegistry) {
+    # Environment variable not found. Try to set it using demo team function.
+    Set-DemoEnvironmentVariables
+    refreshenv
+  }
+
+  # Retry
+  $SitecoreRegistry = $env:INTERNAL_SITECORE_DOCKER_REGISTRY
+  if ($null -eq $SitecoreRegistry) {
+    # Environment variable not found. Use public Sitecore Docker registry.
+    $SitecoreRegistry = "scr.sitecore.com/"
+  }
+}
+
 $NanoserverVersion = $(if ($WindowsVersion -eq "ltsc2019") { "1809" } else { $WindowsVersion })
 
 Set-DockerComposeEnvFileVariable "SITECORE_DOCKER_REGISTRY" -Value $SitecoreRegistry
```

Comment:
Ignore setting SitecoreRegistry and DockerRegistry in the env file if either of those files are null or empty (not passed in to script or not found in .env)